### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by nang-dev**
change off of rubberduck foundation

### CODE REVIEW

**Review by Fryingpannn — APPROVED** [review]
No Ducks Allowed

File: lib/extension/src/conversation/template/parsePearAITemplate.ts [hunk 1]
```diff
@@ -76,7 +80,9 @@ export function parsePearAITemplate(
 
     const templateText = namedCodeSnippets.get("json conversation-template");
 
-    const template = pearaiTemplateSchema.parse(secureJSON.parse(templateText));
+    const template = rubberduckTemplateSchema.parse(
```
Comment (Fryingpannn) [inline lib/extension/src/conversation/template/parsePearAITemplate.ts:83]: Still got sum ducks here

File: lib/extension/src/conversation/template/PearAITemplate.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ const variableSchema = zod.discriminatedUnion("type", [
 
 export type Variable = zod.infer<typeof variableSchema>;
 
-export const pearaiTemplateSchema = zod.object({
+export const rubberduckTemplateSchema = zod.object({
```
Comment (Fryingpannn) [inline lib/extension/src/conversation/template/PearAITemplate.ts:104]: ducky

Comment (nang-dev) [reply lib/extension/src/conversation/template/PearAITemplate.ts:104]: this is very strange

**Pull request merged**